QuizWhois.Api/Controllers/QuestionController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using QuizWhois.Domain.Services.Interfaces;
+using Microsoft.AspNetCore.Http;
+using QuizWhois.Domain.Database;
+using QuizWhois.Domain.Entity;
+using QuizWhois.Common.Models;
+
+namespace QuizWhois.Api.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class QuestionController : Controller
+    {
+        private readonly IQuestionService _questionService;
+        private readonly ApplicationContext _applicationContext;
+
+        public QuestionController(IQuestionService questionService, ApplicationContext applicationContext)
+        {
+            _questionService = questionService;
+            _applicationContext = applicationContext;
+        }
+
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [HttpGet]
+        [Route("getall")]
+        public JsonResult Get()
```

Comment:
If method returns all the entries, it would be better to name it appropriately, for example "GetAll", or even create a subset of methods to retrieve data: get/all, get/paged, get/{id} or so

Reply:
Can we also move retrieving data to QuestionService in case we want to apply some logic in the process, overall it's not a good decision to call db straight from controller

Reply:
Let's remove the method for now, it is out of the scope anyways